csrc/ops/arith.cpp

```diff
@@ -74,6 +74,16 @@ TensorView* bitCastOp(DataType dtype, TensorView* v1) {
   return bitCastOp(dtype, v1->as<Val>())->as<TensorView>();
 }
 
+Val* set(Val* v) {
```

Comment:
This is a little off-topic but why is `set` in `arith.cpp` but `permute` is in `alias.cpp`? Especially now that they both just work via setting an "untransformed" rfactor domain (a "transpose-like" domain?), do they belong in alias?

Reply:
This is a good point, let me move it to `alias.cpp`.